src/anyp/Uri.cc

```diff
@@ -1008,56 +1008,54 @@ AnyP::Uri::Uri(AnyP::UriScheme const &aScheme) :
 }
 
 // TODO: fix code duplication with AnyP::Uri::parse()
-char *
-AnyP::Uri::cleanup(const char *uri)
+SBuf
+AnyP::Uri::Cleanup(const SBuf &uri)
 {
-    char *cleanedUri = nullptr;
+    // 'whitespace' for chop and strip are different
+    static const CharacterSet wspChop = (CharacterSet::WSP +
+                                         CharacterSet::LF +
+                                         CharacterSet::CR).rename("wsp-chop");
+#if 0 // XXX: use these instead of xisspace() character match (it varies)
+    static const CharacterSet wspStrip = (CharacterSet("VT,FF","\v\f") +
+                                          wspChop).rename("wsp-strip");
+#endif
```

Comment:
Is there any specific purpose to leaving this dead code in?

Reply:
Intention was to make this PR less controversial by minimizing behaviour change. Fixing this oddity/bug would be a rather big and somewhat silent change to behaviour in an unknown/large set of cases.